lib/media/streaming_engine.js

```diff
@@ -1240,7 +1240,13 @@ shaka.media.StreamingEngine = class {
       if (this.config_.lowLatencyMode && isReadableStreamSupported && isMP4 &&
           !reference.hlsAes128Key) {
         let remaining = new Uint8Array(0);
+        let processingResult = false;
+        let callbackCalled = false;
         const streamDataCallback = async (data) => {
+          if (processingResult) {
```

Comment:
I see the logic here, that these two variables ensure that we will still append if streamDataCallback is not supported, and also ensure that we won't append twice if it is.
It could use comments though.

For this line, I'd suggest something like:
```js
// If the fallback result processing was triggered, don't also append the buffer here.
```

Reply:
Done!